packages/ui/src/components/organisms/DynamicForm/components/custom-inputs/CountryPicker/CountryPicker.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  RJSFInputAdapter,
+  RJSFInputProps,
+  TextInputAdapter,
+} from '@/components/organisms/DynamicForm/components/RSJVInputAdaters';
+import { getCountries } from '@/components/organisms/DynamicForm/components/custom-inputs/CountryPicker/helpers';
+import { FieldProps } from '@rjsf/utils';
+import { useMemo } from 'react';
+
+export type CountryPickerProps = RJSFInputProps<FieldProps<string> & { language?: string }>;
+
+export const CountryPicker: RJSFInputAdapter<string> = props => {
+  const { schema, language = 'en' } = props;
```

Comment:
```suggestion
export const CountryPicker: RJSFInputAdapter<string> = ({ schema, language = 'en' }) => {
```

Reply:
P3

Reply:
Pay attention that after destructurization of props its passed down to input itself. So by using your suggestion ill have to recunstruct props again and pass down.